app/jobs/remote_service_notifier_job.rb

```diff
@@ -25,8 +25,18 @@ class RemoteServiceNotifierError < StandardError
 
   def self.enqueue(object_id, object_class, action)
     job = new(object_id, object_class, action)
+    should_queue = true
 
-    Delayed::Job.enqueue job, queue: 'remote_service_notifier'
+    if ((object_class == 'Study') || (object_class == 'Project'))
```

Comment:
Could you add a brief comment into the code for documentation in case someone stumbles on this in the future?